cmake/FindPyQt6.py

```diff
@@ -48,7 +48,7 @@
 print("pyqt_version_str:%s" % PyQt6.QtCore.PYQT_VERSION_STR)
 print("pyqt_mod_dir:%s" % os.path.join(cfg.default_mod_dir, "PyQt6"))
 print("pyqt_sip_dir:%s" % sip_dir)
-print("pyqt_sip_flags:%s" % PyQt6.QtCore.PYQT_CONFIGURATION['sip_flags'])
+print("pyqt_sip_flags: ")
```

Comment:
Why removing the printed value?

Reply:
There is a dedicated commit for this :sweat_smile:: `PYQT_CONFIGURATION` does not exist any more in Qt6

Reply:
Can we remove the print so ?

Reply:
as described [here](https://www.riverbankcomputing.com/static/Docs/PyQt6/pyqt5_differences.html) PYQT_CONFIGURATION has indeed been removed. I would be in favor to remove the print() and the subsequent modification in cmake file